src/main/java/bhs/devilbotz/commands/DriveSetDistancePID.java

```diff
@@ -4,20 +4,31 @@
 
 package bhs.devilbotz.commands;
 
+import bhs.devilbotz.Constants;
 import bhs.devilbotz.subsystems.DriveTrain;
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import edu.wpi.first.wpilibj2.command.CommandBase;
 
+/**
+ * This command is a PID controller that drives the robot to a set distance. (Command does not check
+ * if robot is going straight)
+ */
 public class DriveSetDistancePID extends CommandBase {
   private DriveTrain drivetrain;
   private PIDController distance_pid;
```

Comment:
```suggestion
  private PIDController distancePid;
```
mismatching variable names preventing build (change to match the declaration)